src/robot_gnss/skeleton.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# СКЕЛЕТ ОСНОВНОЙ ПРОГРАММЫ
```

Comment:
It is more correct to use the word «structure» instead of the word «skeleton».

Reply:
https://en.wikipedia.org/wiki/Skeleton_(computer_programming)  
I will allow myself to refer to Wikipedia. The script is intended to explain the approximate code of what needs to be written.